backend/model/Users.js

```diff
@@ -139,6 +139,7 @@ const UsersAPI = {
   },
 
   uploadUserProfilePicture: async function (uid, file) {
+    console.log('here');
```

Comment:
I believe that whatever message is logged should be a little bit more descriptive.